### PR TITLE
Removes portal storm event

### PR DESCRIPTION
**Pull request by Time-Green**
Alternative to: #447

The only way to fix this is to remove this.

Sorry fluffy.

:cl: God
bugfix: Removes portal storm. It's existence is a bug, to humanity.
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
:-1: This event is perfectly fine, the nerfs in the other PR are fine.

Why not fix the issue with the earliest_start not functioning?

**Comment by Nirnael** [comment]
> Why not fix the issue with the earliest_start not functioning?

thats already fixed
also
M E M E P U L L R E Q U E S T

File: code/modules/events/portal_storm.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/portal_storm_syndicate
 	name = "Portal Storm: Syndicate Shocktroops"
 	typepath = /datum/round_event/portal_storm/syndicate_shocktroop
-	weight = 2
+	weight = 0 //Set to 1 or 2 to enable, you monster
```
Comment (Nirnael) [inline code/modules/events/portal_storm.dm]: I already explained in the other PR that weight 0 doesn't disable the event. Next one to PR this gets a permaban from github I swear :kappa:

File: code/modules/events/portal_storm.dm [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	name = "Portal Storm: Syndicate Shocktroops"
 	typepath = /datum/round_event/portal_storm/syndicate_shocktroop
 	weight = 0 //Set to 1 or 2 to enable, you monster
-	min_players = 15
+	min_players = 666 //To make sure it never starts
```
Comment (Nirnael) [inline code/modules/events/portal_storm.dm]: REEEEEEEE read the other PR, it's max_occurences to 0.

**Comment by Super3222** [comment]
This is a terrible way to remove it. Try commenting out the entire event or make the weight 0.

**Comment by Time-Green** [comment]
I just commented it out instead...

**Comment by Super3222** [comment]
Alright two things.
1. I was wrong. Don't comment it out. Place weight as 0.
2. Never make a commit message like "this is a good thing" ever again.

**Comment by ShadowDeath6** [comment]
confirmed super gets buttmad about commits

also this is actually a good thing, pls remove event

**Comment by ghost** [comment]
It's an uncounterable round ruining event. Unlike kudzu, blob, meteor, ect-which you can patch up the damage and survive-20 nukeops spawn and kill everyone. Even /tg/station, which is way more powergamey than us, disabled it for a good reason.

It sither gives everyone nukeop gear-including elite hardsuits and eshields-or kills 60% of the station and ends the round. It's useless bullshit.

This is not 'adapting'.

Before I went on LoA I reminded all admins if this event should trigger to delete the mobs when they spawned. I am quite frankly annoyed that they forgot this.

The event is not 'fun'. It kills a bunch of unarmed people, gives whatever antag of the day shittons of overpowered gear (eshields on shadowlings, elite hardsuit + carbine changelings) and altogether doesn't fit with yogs.

Disregarding fluffy's usual useless drivel, the event should be disabled unless admin-forced.

**Comment by FluffySurvivor** [comment]
Oh no you can't powergame your job all round and you end up getting killed what a tragedy let's remove a rare and challenging event so I'll never have to leave my 5x5 room ever again! @AdamElTablawy

I literally killed one with a hatchet by dodging the bullets, they are not hard to kill. Just group on one and it's easy

**Comment by Jc1469** [comment]
Salt salt salt.  

Don't disable shit because of salt.  
If the required time for random events is fixed, less salt, less bitching, more syndicates killing people with less salt.

**Comment by ghost** [comment]
@Oisin100 merge.

Otherwise I can just delete the syndie mobs whenever they spawn when I get back-It's within my power and the admin rules to do so unless there's some invisible rule about it, seeing as admins can force-delete things that would otherwise destroy the round (see: xenos/randomblob)

**Comment by GrayRachnid** [comment]
> @Oisin100 merge.
> Otherwise I can just delete the syndie mobs whenever they spawn when I get back-It's within my power and the admin rules to do so unless there's some invisible rule about it, seeing as admins can force-delete things that would otherwise destroy the round (see: xenos/randomblob)

You don't threaten a headcoder with abusing your powers adam, and this does a have rule.

> Deleting an object, structure or player/mob for no legitimate reason

https://forums.yogstation.net/index.php?threads/admin-code-of-conduct.1478

**Comment by ghost** [comment]
There is legit reason tho.

If they show up during a round they can irreversibly fuck it up, just like xenos/randomblob/looping immovable rods, all of which can be deleted if the admin believes it to be beneficial to the round.

If it shows up during a boring round, sure, but if it fires during blob/ops/wiz/shadowling or whatever i'll delete them because it's shitty for both crew and antag to be playing a fun round and then suddenly carbine'd by 1300 ops at the same time-especially if you're a nukeop and get killed by the syndiestorm portal. Imagine the salt.

And i'm not saying 'merge or else' but I can see how you would think that I was directing the second part at oisin. I'm saying merge to get oisin's attention and the rest is completely unrelated to me flagging him over-albiet that could be phrased better.

**Comment by Nirnael** [comment]
Make it so that when the syndies die they activate their bomb implant and gib themselves so no loot is left, problem solved.

> There is legit reason tho.
> 
> If they show up during a round they can irreversibly fuck it up, just like xenos/randomblob/looping immovable rods, all of which can be deleted if the admin believes it to be beneficial to the round.
> 
> If it shows up during a boring round, sure, but if it fires during blob/ops/wiz/shadowling or whatever i'll delete them because it's shitty for both crew and antag to be playing a fun round and then suddenly carbine'd by 1300 ops at the same time-especially if you're a nukeop and get killed by the syndiestorm portal. Imagine the salt.
> 
> And i'm not saying 'merge or else' but I can see how you would think that I was directing the second part at oisin. I'm saying merge to get oisin's attention and the rest is completely unrelated to me flagging him over-albiet that could be phrased better.

I don't think any admin has deleted xeno, random blob or rod until now. I mean it's hard but the shuttle gets called anyway and most antags don't get bothered by it.

**Comment by ghost** [comment]
They have in the past. I whined at 20th for deleting an immovable rod that was looping but it IS allowed afaik. Also used to be standard policy to delete xenos but that stopped. Randomblob got del'd a couple days before I went on LoA.

Also, the issue isn't just the loot, it's how absurdly lethal the mobs are. I'd be fine if the majority of the spawned syndies were the punch ones in swat gear lead by one in an elite hardsuit, or numbers reduced, but i'd prefer if the event was left as admin only.

**Comment by ghost** [comment]
Actually, why not just leave the event as is and have it fire at the 2hr mark or so with a high chance, since that's when admins force-call the shuttle?

**Comment by Jc1469** [comment]
> Actually, why not just leave the event as is and have it fire at the when the round is over, effectively removing it like I want to anyways?

Someone already fixed the required time before random events thing. It's just not been merged/updated yet.  
There's nothing wrong with this, blob, anything. Just the fucking timers being broken.  If the crew can't deal with a blob ~30 minutes into a round, they deserve what they get. Same goes for this.

**Comment by ghost** [comment]
Hey, slaughter demons also randomfire at the 2hr mark, afaik. Tg might have reverted that. And they're less deadly than this

Or blacklisting it from firing during blob, shadowling, cult, nukeops or clock cult would work. Those are the modes i'm worried about it firing in. The other ones crew can deal with, but blob + carbinespam...

**Comment by X-TheDark** [comment]
Also, if I remember correctly, auto-call the shuttle at about 1h 45-50m mark, not at 2h mark, so you'd literally get 3 minutes of your "event" at best.

**Comment by Stealthkibbler** [comment]
imho we should replace it with an anomaly that summons a laughter demon that has to tickle the clown to death or something. Would lead to much funnier scenarios, you cant go wrong with clowns.

**Comment by ShadowDeath6** [comment]
slaughter demon event when

**Comment by Zostroll** [comment]
I would like to say that I happy to remove the event if it's what people want, when I originally started my pr it was to remove the event but I was encourage to fix it. If the consensus is to remove it I will change my pr. The feedback I have received is very mixed.

**Comment by Super3222** [comment]
IF you're sticking with removing the event:

Commenting it out means that admins can't fire it themselves @timkoster1 Lower the weight to 0. It would only fire on a super, super, super rare chance. Maybe if you let the same round play over for a span of 24 hours you'd have a chance of it firing like the event that spawns a lone op in space with full gear and a radio uplink.

**Comment by killerx09** [comment]
Alternative idea

We have a unwritten rule about ending the round at the two hour mark

Sometimes there's no admins on to force the shuttle call, extending the round slightly further

Sometimes the players don't call the shuttle after the two hour mark

Soo, how about in the astronomical case that both of those happen, syndicate mobs spawn and give a good reason to finally call the shuttle?

(Etc jack the weight up, but put the earliest time to something like fifteen minutes after the two hour mark.)

**Comment by Jc1469** [comment]
Sometimes-inactive admins isn't a reason to change code.
Salt isn't a reason to change code.
@Silk-Touch I don't even know who you are, but the general consensus on Git doesn't mean shit.  A single poll on the server doesn't mean shit.  Heck, a poll on the forum doesn't really mean shit because a lot of our players don't use the forums.

# We don't remove things because they're overpowered. We nerf them.

## And raising the minimum spawn time or 2 hours isn't a nerf. It's removal.

# **You're coders. You don't decide what is and isn't kept.**

Holy fucking shit.

**Comment by sagattyy** [comment]
Sorry @Jc1469 this is yogstation we can't nerf things only remove
"The only way to fix this is to remove this."

**Comment by Nirnael** [comment]
> @Jc1469 bla

That's how it has always been done, tg does it, everyone does it. You PR it, if it's controversial it's closed, if not it's allowed and it's tested. The players that don't use media get fucked and test it live, if they don't like it they are free to complain.

**Comment by george99g** [comment]
I'm just going to close this. For one, this PR, the commits and the changelog entry are abysmal.
The other reason is that this event can be salvaged, one way or another. No need to remove content.

**Comment by ShadowDeath6** [comment]
Just because it can be salvaged doesn't mean it should, we already have carp and spider events to kill people, we don't need to have more event spam shit that gets people killed, especially when the mobs are twice as strong as any carp or spider

**Comment by Super3222** [comment]
> You're coders. You don't decide what is and isn't kept.

Since the (dawn of time) moment Yogstation's code went public, we've been doing that. And as Nirnael puts it, it's how almost every other SS13 repo does it as well.

We are the flow of what comes in and out of the code, that is literally how it works. The general consensus on a PR is what, most of the time, drives the code maintainers / HC's / Council to merge or close PR's. Opinions do matter, the voices of people who care what comes in and out do matter. Things get pointed out, discussions stir about, and the people that actually care will throw in their say.

And regarding this PR, I was, and still am, neutral. I'm glad a decision has finally been made though, despite how redundant this event is.

With the recent fix to the config, this event will fire 30 minutes into the round instead of a minute.
And with https://github.com/yogstation13/yogstation/pull/447 nerfing some of their immunity, we can all play 99% stress free.

**Pull request closed without merging**